aptos-move/e2e-benchmark/src/main.rs

```diff
@@ -139,8 +139,8 @@ fn main() {
         (26, EntryPoints::ResourceGroupsSenderMultiChange {
             string_length: 1024,
         }),
-        (287, EntryPoints::TokenV1MintAndTransferFT),
-        (458, EntryPoints::TokenV1MintAndTransferNFTSequential),
+        (350, EntryPoints::TokenV1MintAndTransferFT),
+        (536, EntryPoints::TokenV1MintAndTransferNFTSequential),
```

Comment:
@igor-aptos expected?

Reply:
seems like this regression was committed today  - https://aptoslabs.grafana.net/goto/ePHFS7ESg?orgId=1, so fine from this PRs perspective